ericberringer/currency-kidz-react
# Ck quiz

Type of change:

Testing Instructions

/quiz. Wrong answers should give you a visual prompt to try again and a correct answer should give you a clear passing prompt.

git fetch --all 
git checkout CK-Quiz

Checklist:

X(I have preformed a self review)

X(Code follows style guidelines of project)

(I have commented my code. Especially w/ hard to understand code)

X(Changes I have made generate no new warnings or errors)

X(I have added test instructions that prove my fix is effective or that my feature works)

(I am not ruminating about the things my therapist told me to forget about b/c I cannot do anything about them. For example, the state of the world. )

- Fixes #
